applications/FluidDynamicsApplication/custom_strategies/strategies/residualbased_simple_steady_scheme.h

```diff
@@ -168,7 +170,7 @@ class ResidualBasedSimpleSteadyScheme : public Scheme<TSparseSpace, TDenseSpace>
     if (SteadyLHS.size1() != 0)
       noalias(LHS_Contribution) += SteadyLHS;
 
-    AddRelaxation(rCurrentCondition->GetGeometry(), LHS_Contribution, RHS_Contribution, CurrentProcessInfo);
+    // AddRelaxation(rCurrentCondition->GetGeometry(), LHS_Contribution, RHS_Contribution, CurrentProcessInfo);
```

Comment:
If we go with the proposed approach, shouldn't be directly remove the `AddRelaxation` function? (also, better remove dead code instead of commenting it out).